d3/query-graphs.css

```diff
@@ -6,7 +6,6 @@ html, body {
 }
 #tree-container {
   padding: 0px;
-  padding-top: 53px;
```

Comment:
The original motivation for the relatively large top padding was to make space for the hover text of the root node. Without it the root hover text for all of the test queries are a little obscured and most of the root hover text is obscured for the dint queries, e.g., dint4.xml.

I suppose we could make the padding a function of orientation? Even better would be to make hover text always render visibly regardless of padding, but that may be more than we want to take on. 

Just noticed that the spinner is now also chopped off. Challenging to observe because of the fast layout and rendering! Try http://localhost:3000/d3/query-graphs.html?file=hyper-steps2.json&orientation=left-to-right&collapse=n. So this is not only a hover text issue.

Reply:
I didn't notice that regression, thanks for catching it :)
I adapted the last commit such that:
* The root node is positioned a bit lower now, so that the tooltip is visible by default (adjusted orientations["top-to-bottom"].rooty)
* changes CSS such that the spinner is vertically centered

Reply:
Changes look good. Because we no longer have a result node root, which naturally had a small hover text, any non-trivial root node hover text will likely be partially obscured, but that was also already true for large hover text even with a result node. Someday we can improve the location awareness of hove text.